Makefile

```diff
@@ -152,8 +152,43 @@ install-pipelines: ## Install the CNBI pipelines into the configured cluster.
 uninstall-pipelines: ## Uninstall the CNBI pipelines from  the configured cluster. Call with ignore-not-found=true to ignore resource not found errors during deletion.
 	kubectl delete --ignore-not-found=$(ignore-not-found) -f 'hack/cnbi-*.yaml'
 
+define managed_operators
+apiVersion: operators.coreos.com/v1alpha1
+kind: Subscription
+metadata:
+  name: openshift-pipelines-operator
+  namespace: openshift-operators
+spec:
+  channel: latest
+  name: openshift-pipelines-operator-rh
+  source: redhat-operators
+  sourceNamespace: openshift-marketplace
+---
+apiVersion: operators.coreos.com/v1alpha1
```

Comment:
idk if we should trigger the deployment of cert-manager on deploy. 
if we consider `make deploy` a dev time tool its ok.